recipes/conipher/meta.yaml

```diff
@@ -0,0 +1,82 @@
+{% set version = "2.0.0" %}
+{% set sha256 = "aa878da6a50e875befe0e474b19d9412d045bd4a4af898bf48c3c493b1fa404f" %}
+
+package:
+  name: conipher
+  version: '{{ version }}'
+
+source:
+  url: https://github.com/McGranahanLab/CONIPHER/archive/v{{ version }}.tar.gz
+  sha256: '{{ sha256 }}'
+
+build:
+  number: 0
+  noarch: generic
+  rpaths:
+    - lib/R/lib
+    - lib/
+
+requirements:
+  host:
+    - r-base <=3.6.1
```

Comment:
This is a very old R version. Is it not working with newer versions?

Reply:
I've updated the versions, but due to some incompatibilities it does not work on the newest r versions for now